README.md

```diff
@@ -11,10 +11,15 @@ Enhanced History for Chrome
 
 Chrome is an excellent browser, but browsing your history kinda stinks.
 
-Getting rolling
+Manual Installation
 ----------------
 
-1. Clone this repo
-2. Install the repo's directory as an extension
+1. Clone this repo locally or download and unzip
+3. To Install the repo's directory as an extension open the Extension Management page by navigating to chrome://extensions.
+   - The Extension Management page can also be opened by clicking on the Chrome menu, hovering over More Tools then selecting Extensions.
+4. Enable Developer Mode by clicking the toggle switch next to Developer mode.
+5. Click the LOAD UNPACKED button and select the extension directory.
 
-Any changes you make to your cloned repo will be seen immediately in the installed extension. Though sometimes changing translation files (in _locales/) requires a manual extension refresh in Chrome's extension settings.
+![Select Directory and click LOAD UNPACKED](https://i.imgur.com/blokl7A.png)
```

Comment:
Instead of imgur, can you put the image in the repo and reference it that way?

Ie:

```
![Select ...](docs/manual-install-chrome-load-unpacked.png?raw=true)
```

Reply:
Added https://github.com/chrome-enhanced-history/enhanced-history/pull/13/commits/7d6130521738694e175c0f64bd089189ecaaf2a3 and https://github.com/chrome-enhanced-history/enhanced-history/pull/13/commits/f9d86e427611ea35a3cff41901b957d7aa4943b8 to fix this image linking issue